src/internal/_assocPath.js

```diff
@@ -0,0 +1,11 @@
+var _assoc = require('./_assoc');
+var _slice = require('./_slice');
+
+
+module.exports = function _assocPath(path, val, obj) {
```

Comment:
`undefined` is what you get for `path('x', {y: 1})`, yes? Seems like the same situation, analogous to `find`. Another _unsafe_ function, to be sure